src/bin/pg_autoctl/ipaddr.c

```diff
@@ -375,8 +376,7 @@ fetchIPAddressFromInterfaceList(char *localIpAddress, int size)
 static bool
 isTestEnv(void)
 {
-	const char *socketDir = getenv("PG_REGRESS_SOCK_DIR");
-	return socketDir != NULL && socketDir[0] == '\0';
+	return get_env_variable("PG_REGRESS_SOCK_DIR", NULL, 0) == 0;
```

Comment:
Could maybe be made a function too (is done 3 times), e.g. `bool unix_sockets_enabled()`